app/src/sharedTest/java/org/oppia/app/parser/StringToRatioParserTest.kt

```diff
@@ -53,29 +49,52 @@ class StringToRatioParserTest {
   }
 
   @Test
-  fun testParser_submitTimeError_returnInvalidSizeError() {
+  fun testParser_submitTimeError_numberOfTermsZero_returnValid() {
+    val error = stringToRatioParser.getSubmitTimeError("1:2:3:4", numberOfTerms = 0)
+      .getErrorMessageFromStringRes(context)
+    assertThat(error).isEqualTo(null)
+  }
+
+  @Test
+  fun testParser_submitTimeError_numberOfTermsThree_returnInvalidSizeError() {
+    val error = stringToRatioParser.getSubmitTimeError("1:2:3:4", numberOfTerms = 3)
+      .getErrorMessageFromStringRes(context)
+    assertThat(error).isEqualTo("Number of terms is not equal to the required terms.")
+  }
+
+  @Test
+  fun testParser_submitTimeError_numberOfTermsFour_returnInvalidSizeError() {
```

Comment:
Test name does not match outcome/conditions. Please fix. (I'm guessing you meant to pass in a three-component ratio).